### ISSUE TITLE
[Bug]: Fork option buttons and checkboxes not accessible via screen reader

### ISSUE THREAD

**Issue by gmerritt** [danny-avila/LibreChat#7144]
### What happened?

The buttons and checkboxes and tool tips of the fork options are not accessible by screen reader navigation; attached screen recording with audio reviews the difficulties.

### Version Information

As of [this commit](https://github.com/danny-avila/LibreChat/commit/7f1d01c35a7c9a916dc6d33f9f3915b4a452b4ad) -- April 23, 2025

### Steps to Reproduce

1. Access fork options with screen reader

### What browsers are you seeing the problem on?

Chrome, Firefox

### Relevant log output

```shell
Note attached screencast upload.
```

### Screenshots

UC Berkeley [Web Access Analyst Lucy Greco](https://www.berkeley.edu/directory/?search-term=Greco,%20Lucy&uid=264621) may be contacted if additional information is needed.

https://github.com/user-attachments/assets/1d510343-64b4-426c-a026-4b47001d95bc

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct
